way_to_home/.pylintrc

```diff
@@ -7,7 +7,7 @@ extension-pkg-whitelist=
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=CVS, migrations, apps.py, settings.py, local_settings.py, manage.py, unittests, urls.py
+ignore=CVS, migrations, apps.py, settings.py, local_settings.py, manage.py, unittests, urls.py, bundle.js, static
```

Comment:
remove bundle.js because of it contained in the static folder

Reply:
fixed